### ISSUE TITLE
Step-by-step navigation 'or' headings must be nested under their parent step headings

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-frontend#3677]
This issue is from May 2023 external accessibility audit report.
**WCAG Reference:**
1.3.1 Info and Relationships (Level A)
[Understanding Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) | [How to Meet Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)

**Issue ID:**
DAC_Information_And_Relationships_01

**URLs:**
https://www.gov.uk/visit-uk-business-trip

## Screen shots
<img width="646" alt="image" src="https://github.com/alphagov/govuk-frontend/assets/98318778/57bae294-46ce-45ef-9cc6-1ba496b00859">

The step by step navigation page consists of an ordered list, informing users of screen reading assistive technologies that the content is presented in a step by step process and a specific order. However, the list items have been provided with ‘list style: none;’ meaning that the steps have been removed from the list and rely on the heading text instead.
The two ‘Or’ steps have been marked up as level 2 headings as well as the other headings, which means that they do not programmatically relate to the previous step to which they offer the alternative for.

## Current code ref(s)
`#apply-for-a-family-permit`

```<li class="gem-c-step-nav__step js-step" id="apply-for-a-visa" data-track-count="stepNavSection">
<div class="gem-c-step-nav__header js-toggle-panel" data-position="3">
<h2 class="gem-c-step-nav__title"> <span class="gem-c-step-nav__circle gem-c-step-nav__circle--number">
<span class="gem-c-step-nav__circle-inner">
<span class="gem-c-step-nav__circle-background">
<span class="gem-c-step-nav__circle-step-label govuk-!-display-none-print visuallyhidden">Step</span> 3<span class="gem-c-step-nav__circle-step-colon govuk-!-display-none-print visuallyhidden" aria-hidden="true">:</span>
</span>
</span>
</span>
[…]
</li>
<li class="gem-c-step-nav__step js-step" id="apply-for-a-family-permit" data-track-count="stepNavSection">
<div class="gem-c-step-nav__header js-toggle-panel" data-position="4">
<h2 class="gem-c-step-nav__title">
<span class="gem-c-step-nav__circle gem-c-step-nav__circle--logic">
<span class="gem-c-step-nav__circle-inner">
<span class="gem-c-step-nav__circle-background">
or
</span>
</span>
</span>
[…]
</li>
```

## Tester comments
No comments recorded.

## Solution
Ensure that all information and relationships of content are equally conveyed to users of screen reading assistive technologies as they are presented to other users visually. This will help the user understand the structure of content and how it relates.
As the ‘Or’ steps are not the sequential steps following 1,2, and 3, they should not be presented as such programmatically for screen reader users.
Consider nesting the ‘Or’ steps as sub-headings of step 3 as level 3 headings in the list. This will convey that the ‘or’ steps are not the next steps in the sequential order.

**Comment by davidc-gds** [comment]
This issue has been assessed to not be a high-severity accessibility concern. It doesn't appear to meet any of our 3 [criteria for a high-severity accessibility concern](https://design-system.service.gov.uk/community/accessibility-strategy/#determining-severity-of-an-accessibility-concern).

- Despite reasonable evidence that it fails a WCAG criterion, there is no evidence that this makes it difficult or impossible for some people to complete certain tasks. ❌
The evidence of a WCAG failure:
    - It is related to WCAG level A criterion: [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
    - It was assessed in an external accessibility audit as a 'high' priority accessibility concern.
- It does not appear to affect the accessibility of essential services or critical infrastructure. ❌
- There is no blocker to individual service teams efficiently resolving it on their own. ❌

As such, this will not be flagged as a high-priority issue.

We will soon reach out to the team that manages the '[Visit the UK on a business trip](https://www.gov.uk/visit-uk-business-trip)' service, to inform them of the issue.

In the coming weeks we will investigate whether it fails WCAG level A criterion: [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
